src/controllers/controller.bubble.js

```diff
@@ -135,6 +135,14 @@ module.exports = DatasetController.extend({
 		model.radius = options.radius + options.hoverRadius;
 	},
 
+	/**
```

Comment:
How come we don't need this in the line and radar controllers?

Reply:
Only bubble overrides `_resolveDataElementOptions`

Reply:
Right, the other ones are `_resolveDatasetElementOptions`.